packages/react-devtools-shared/src/devtools/views/Profiler/CommitFlamegraphListItem.js

```diff
@@ -43,6 +46,23 @@ function CommitFlamegraphListItem({data, index, style}: Props) {
     [selectFiber],
   );
 
+  const handleMouseOver = useCallback(
```

Comment:
These `useCallback` for mouse handlers can be removed :)

Reply:
Done!